src/stirling/e2e_tests/stirling_wrapper_size_test.cc

```diff
@@ -30,7 +30,7 @@ namespace stirling {
 #ifdef __OPTIMIZE__
 constexpr uint64_t kFileSizeLimitMB = 118;
 #else
-constexpr uint64_t kFileSizeLimitMB = 295;
+constexpr uint64_t kFileSizeLimitMB = 297;
```

Comment:
I think we should raise this by 5 MiB (300) to provide more of a buffer. While your change tripped the threshold any significant code added in stirling will inch us closer to this (adding libbson, the mongo parser/stitcher, which includes templates, etc). So while your change added more code and templates instantiations, I think it's likely that the mongo work has contributed to this as well.

Reply:
I've opened a separate PR for this #1731